### PR TITLE
Pricing page: implement most popular experiment

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR implements the A/B experiment that replaces the plan cards row by the most popular products, in the pricing page. It also makes the ascending prices experiment the new default.

Fixes 1196341175636977-as-1199508074334637

### Implementation notes

Implementation isn't clean. Fortunately the lifespan of the experiment is quite short.

### Testing instructions

_Prerequisite: you'll need to change the variation of the `popularProductsOffering` experiment shown in the capture below:_
Screen Shot 2020-12-15 at 10 15 44 AM

- Download the PR
- Run Calypso or Jetpack cloud
- Visit the pricing page
- Check that you see the new default page. Order of the plan cards row should now be: Security Daily, Security Real-time, Complete
- Switch to the `dailyProducts` variation
- Check that the order of cards, the 2 section headings, and features (in the top row only) match the copy and mockups (see linked task, and captures below). The case of the features strings can be different to be consistent with the rest of the page.

### Screenshots

_control_
Screen Shot 2020-12-15 at 10 16 04 AM
Screen Shot 2020-12-15 at 10 16 11 AM

_dailyProducts_
Screen Shot 2020-12-15 at 10 15 13 AM

Screen Shot 2020-12-15 at 10 15 23 AM

### CODE REVIEW

File: client/lib/plans/features-list.js [hunk 1]
```diff
@@ -1073,14 +1080,20 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_BACKUP_DAILY_V2 ]: {
 		getSlug: () => constants.FEATURE_BACKUP_DAILY_V2,
-		getTitle: () => i18n.translate( 'Automated daily site backups' ),
+		getTitle: () =>
+			POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+				? i18n.translate( 'Automated daily backups (off-site)' )
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js:1085]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Daily Off-site Backups' )` *ES Score: 6*

File: client/lib/plans/features-list.js [hunk 2]
```diff
@@ -1073,14 +1080,20 @@ export const FEATURES_LIST = {
...
 	},
 
 	[ constants.FEATURE_BACKUP_REALTIME_V2 ]: {
 		getSlug: () => constants.FEATURE_BACKUP_REALTIME_V2,
 		getTitle: () =>
 			( {
-				i5: i18n.translate( 'Automated real-time backups' ),
+				i5:
+					POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+						? i18n.translate( 'Backup (real-time, off-site)' )
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js:1095]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Backup Real-time (off-site)' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/pricing-page-ab-test)

File: client/lib/plans/features-list.js [hunk 3]
```diff
@@ -1212,7 +1228,10 @@ export const FEATURES_LIST = {
 		getIcon: () => ( { icon: 'security', component: MaterialIcon } ),
 		getTitle: () =>
 			( {
-				i5: i18n.translate( 'Scan Daily (automated)' ),
+				i5:
+					POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+						? i18n.translate( 'Scan (daily, automated)' )
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js:1233]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/lib/plans/features-list.js [hunk 4]
```diff
@@ -1240,7 +1259,10 @@ export const FEATURES_LIST = {
 		getIcon: () => ( { icon: 'security', component: MaterialIcon } ),
 		getTitle: () =>
 			( {
-				i5: i18n.translate( 'Real-time Scan (automated)' ),
+				i5:
+					POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+						? i18n.translate( 'Scan (real-time, automated)' )
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js:1264]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Automated real-time scanning' )` *ES Score: 10*

File: client/lib/plans/features-list.js [hunk 5]
```diff
@@ -1263,7 +1285,10 @@ export const FEATURES_LIST = {
 		getSlug: () => constants.FEATURE_ANTISPAM_V2,
 		getTitle: () =>
 			( {
-				i5: i18n.translate( 'Always-on spam protection' ),
+				i5:
+					POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+						? i18n.translate( 'Anti-spam ' )
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js:1290]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **37** times:
`translate( 'Anti-Spam' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/pricing-page-ab-test)

File: client/my-sites/plans/jetpack-plans/products-grid-i5/index.tsx [hunk 6]
```diff
@@ -128,7 +146,11 @@ const ProductsGridI5: React.FC< ProductsGridProps > = ( {
 	return (
 		<Experiment name={ SWITCH_PLAN_SIDES_EXPERIMENT }>
 			<section className="products-grid-i5__section">
-				<h2 className="products-grid-i5__section-title">{ translate( 'Product Bundles' ) }</h2>
+				<h2 className="products-grid-i5__section-title">
+					{ POPULAR_PRODUCTS_OFFERING_VARIANT === tracksVariation
+						? translate( 'Most Popular' )
```
Comment (a8ci18n) [inline client/my-sites/plans/jetpack-plans/products-grid-i5/index.tsx:151]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Most popular' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/pricing-page-ab-test)

**Review by elliottprogrammer — APPROVED** [review]
It works well. Looks good.
LGTM! 👍

**Pull request merged**